src/components/Seedphrase.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function Seedphrase({ seedphrase, isBlurred = true }) {
+  return (
+    <div className="seedphrase slashed-zeroes d-flex flex-wrap">
+      {seedphrase.split(' ').map((seedWord, index) => (
+        <div key={index} className="d-flex py-2 ps-2 pe-3">
+          <span className="seedword-index text-secondary text-end">{index + 1}</span>
+          <span className="text-secondary">.&nbsp;</span>
+          <span className={isBlurred ? 'blurred-text' : ''}>{isBlurred ? 'random' : seedWord}</span>
```

Comment:
```suggestion
          <span className={isBlurred ? 'blurred-text' : ''}>{isBlurred ? 'abcdef' : seedWord}</span>
```

Unrelated but while we're touching this I just remembered that we wanted to change this to a word that's not in the BIP39 wordlist.